cmake/linker_script/common/common-ram.cmake

```diff
@@ -96,8 +97,7 @@ if(CONFIG_ZTEST)
 endif()
 
 if(CONFIG_ZBUS)
-  zephyr_iterable_section(NAME zbus_observer GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN CONFIG_LINKER_ITERABLE_SUBALIGN)
-  zephyr_iterable_section(NAME zbus_channel_observation_mask GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN CONFIG_LINKER_ITERABLE_SUBALIGN)
+  zephyr_iterable_section(NAME zbus_channel_observation_mask GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 1)
```

Comment:
Why did you remove the `zbus_observer` section instead of fixing that?

Reply:
It's added in `common-rom.cmake`.

Reply:
I see, sure. It is constant. So, it was wrong indeed. Thanks for that.